tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -619,6 +619,56 @@ describe('util/extension-runners/chromium', async () => {
       }),
   );
 
+  it('does pass default prefs to chrome', async () => {
+    const { params } = prepareExtensionRunnerParams();
+
+    const runnerInstance = new ChromiumExtensionRunner(params);
+    await runnerInstance.run();
+
+    sinon.assert.calledOnce(params.chromiumLaunch);
+    sinon.assert.calledWithMatch(params.chromiumLaunch, {
+      prefs: {
+        extensions: {
+          ui: {
+            developer_mode: true,
+          },
+        },
+      },
+    });
+
+    await runnerInstance.exit();
+  });
+
+  it('does pass custom prefs to chrome', async () => {
+    const { params } = prepareExtensionRunnerParams({
+      params: {
+        customChromiumPrefs: {
+          'download.default_directory': '/some/directory',
+          'extensions.ui.developer_mode': false,
```

Comment:
Let's split the test in two: that a custom pref can be passed, and that a default pref can be overridden.

We want to make sure that custom prefs are merged with the default prefs, and not changed. The current implementation is correct, but the test as constructed would still pass if the implementation failed to merge, which is a sign of an incomplete test.